keepassxc-browser/_locales/en/messages.json

```diff
@@ -104,7 +104,7 @@
         "description": "Checkbox text below the password generator input field."
     },
     "passwordGeneratorBits": {
-        "message": "??? Bits",
+        "message": " Bits",
```

Comment:
Hmm, should not you use proper placeholders for this? See https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Retrieving_message_strings_from_JavaScript

Reply:
The `messages.json` file is not up-to-date. Placeholders are already used. See https://github.com/keepassxreboot/keepassxc-browser/pull/128/files#diff-8ec61d063c3922fa1e1e4d3c60a50be3R129.

Reply:
Actually the line you've linked seems to prepend `???` to `passwordGeneratorBits`, so it actually _does not_ use placeholders… :smile:

Reply:
Yes, you are right about that. I need to add the placeholders to the JS side too. With HTML pages that already works.

Reply:
Fixed :)